### ISSUE TITLE
Sponsored events, marathons

### ISSUE THREAD

**Issue by dboddie** [pybee/paying-the-piper#72]
[Games Done Quick](https://gamesdonequick.com/) is a great example of fundraising for good causes. Perhaps a sponsored marathon/hackathon model might work for short term fundraising efforts.

**Comment by seiyria** [comment]
The question becomes then who will sponsor such a thing, and what sorts of objectives do the sponsors have?

**Comment by dboddie** [comment]
Of course, these are the same questions that are asked when looking for sponsorship for a conference, or when a website runs a campaign to raise funds. For conferences, the sponsorship typically comes from companies who want visibility in relation to a product or project. For websites, it may be that most of the donations come from individuals.

Sponsorship of conferences has often led to sponsorship of sprints hosted at conferences which, while it might be a good way to make progress with a codebase, isn't quite what we might want, which is remuneration for ongoing work on a project. However, hosting fundraisers at conferences might be a way to raise money for developers to work on worthy projects.
